Sources/Amplitude/AMPStorage.m

```diff
@@ -39,7 +40,7 @@ + (NSString *)getAppStorageAmpDir:(NSString *)instanceName {
     NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
     NSString *path = [paths firstObject];
     NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
-    if (instanceName == NULL || [instanceName length] == 0) {
+    if (instanceName == nil || [AMPUtils isEmptyString:instanceName]) {
```

Comment:
Since the instanceName is an object, we should use nil instead of NULL? and I copied how we currently deal with the instanceName  https://github.com/amplitude/Amplitude-iOS/blob/main/Sources/Amplitude/Amplitude.m#L159. @dantetam

Reply:
Yes, it should be nil